olm-catalog/serverless-operator/1.8.0/operator_v1alpha1_knativeserving_crd.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   - JSONPath: .status.conditions[?(@.type=="Ready")].status
     name: Ready
     type: string
-  - JSONPath: .status.conditions[?(@.type=="Ready")].reason
+  - JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
```

Comment:
This diff is fairly large because I opted to just copy upstream's CRD in here for easier management in the future.